test-services/check-links/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+REPO_ROOT_DIR=${1:-"/usr/src/signalfx-agent"}
+
+if [ ! -d "$REPO_ROOT_DIR" ]; then
+    echo "$REPO_ROOT_DIR not found!" >&2
+    exit 1
+fi
+
+if [ ! -f "$REPO_ROOT_DIR/test-services/check-links/config.json" ]; then
+    echo "$REPO_ROOT_DIR/test-services/check-links/config.json not found!" >&2
+    exit 1
+fi
+
+nfailed=0
+for f in $(find "$REPO_ROOT_DIR" -name "*.md" -not -path "$REPO_ROOT_DIR/vendor/*" | sort); do
+    markdown-link-check -c "$REPO_ROOT_DIR/test-services/check-links/config.json" -qv $f
+    (( nfailed += $? ))
+done
+
+[ $nfailed -eq 0 ] || echo -e "\nERROR: dead links found! See output above for details." >&2
+
+exit $nfailed
```

Comment:
Since the max exit code is 255 if there were 256 errors it'd wrap to exit code 0. Unlikely I suppose.

Reply:
> Since the max exit code is 255 if there were 256 errors it'd wrap to exit code 0. Unlikely I suppose.

Good point. I'll just `exit 1` just in case.